Source/HelixToolkit.Wpf.Shared/Importers/StLReader.cs

```diff
@@ -469,8 +469,8 @@ private bool TryReadAscii(Stream stream)
                 }
 
                 line = line.Trim();
-                if (line.Length == 0 || line.StartsWith("\0") || line.StartsWith("#") || line.StartsWith("!")
-                    || line.StartsWith("$"))
+                if (line.Length == 0 || line.StartsWith("\0", StringComparison.Ordinal) || line.StartsWith("#")
+                    || line.StartsWith("!") || line.StartsWith("$"))
                 {
```

Comment:
Consider using:

```csharp
                if (line.Length == 0 || line[0] == '\0' || line[0] == '#' || line[0] == '!' || line[0] == '$')
```

Using `ReadOnlySpan<char>` instead of `string` would also be a nice performance improvement.

Reply:
Done!